### PR TITLE
ECER-1575: Add previous name page

### PR DESCRIPTION
**Pull request by pcatkins**
## Description

- Reset certification store when we fetch certifications list (bug fix)
- Add previous name page

## Related Jira Issue(s)

- [ECER-1575](https://eccbc.atlassian.net/browse/ECER-1575)

## Checklist
- [X] I have tested these changes locally.
- [ ] Changes to backend endpoints are covered by passing integration tests.
- [ ] I have added or updated the necessary documentation.

## Screenshots (if applicable)
Include screenshots to demonstrate the changes visually.

## Additional Comments (optional)
Any additional context or information that might be helpful for the reviewer.

### CODE REVIEW

File: src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/ProfileForm.vue [hunk 1]
```diff
@@ -67,7 +67,7 @@ export default defineComponent({
       if (!valid) {
         this.alertStore.setFailureAlert("You must enter all required fields in the valid format.");
       } else {
-        const success = await putProfile({
+        const { error } = await putProfile({
```
Comment (SoLetsDev) [inline src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/ProfileForm.vue:70]: I think line 83 needs to be changed? I don't see the success variable being declared.

File: src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/pages/AddPreviousName.vue [hunk 2]
```diff
@@ -1,7 +1,156 @@
-<template><div>ADD PREVIOUS NAME</div></template>
+<template>
+  <PageContainer>
+    <v-row>
+      <v-col cols="12">
+        <v-breadcrumbs class="pa-0" :items="items" color="primary">
+          <template #divider>/</template>
+        </v-breadcrumbs>
```
Comment (SoLetsDev) [inline src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/pages/AddPreviousName.vue:7]: I added a common Breadcrumb.vue component. You just add in the list of items that you want :)

File: src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/ProfileForm.vue (hunk 1 above)
Comment (pcatkins) [reply src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/ProfileForm.vue:70]: Yes, thanks bad merge

File: src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/pages/AddPreviousName.vue (hunk 2 above)
Comment (pcatkins) [reply src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/pages/AddPreviousName.vue:7]: Got that with merge too, will do that now

File: src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/pages/AddPreviousName.vue [hunk 3]
```diff
@@ -1,7 +1,156 @@
...
+      );
+    },
+  },
+  methods: {
+    async handleSubmitPreviousName() {
+      // Validate the form
+      const { valid } = await (this.$refs.addPreviousNameForm as typeof EceForm).$refs[previousNameForm.id].validate();
+      if (valid) {
+        //check for duplicate previous name
+        if (!this.duplicatePreviousName) {
```
Comment (SoLetsDev) [inline src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/pages/AddPreviousName.vue:120]: Tough one here... not a big fan of nested if statements. Ideally we could put in a rule somewhere that will check for duplicates. 

in previous-name-form.ts: 
- userStore 
- formStore

then a custom rule in the input ()
- do the loop logic to see if the previous name already exists. 
- if yes, throws an error (doesn't have to give a message)

**Review by SoLetsDev — APPROVED** [review]

**Pull request merged**